src/features/commands/book/book-submission.js

```diff
@@ -0,0 +1,94 @@
+const {
+  ERRORS_DATE_PAST,
+  ERRORS_HOUR_END_BEFORE_START,
+  BOOK_NEAR_END_TEXT,
+  ERRORS_HOUR_END_BEFORE_NOW,
+} = require('../../../utils/texts')
+
+const { SLACK_BOT_USER_OAUTH } = require('../../../../env')
+
+function validate(values) {
+  const errors = {}
+  const now = new Date()
+  now.setHours(0, 0, 0, 0)
+
+  if (new Date(values.date.dateValue.selected_date).getTime() < now.getTime()) {
+    errors.date = ERRORS_DATE_PAST
+  }
+
+  if (
+    parseInt(values.startTime.startTimeValue.selected_option.value) >
+    parseInt(values.endTime.endTimeValue.selected_option.value)
+  ) {
+    errors.endTime = ERRORS_HOUR_END_BEFORE_START
+  }
+
+  if (
+    !errors.endTime &&
+    new Date(values.date.dateValue.selected_date).getTime() <=
+      now.getTime() + 86400000 /*60 * 60 * 24 * 1000*/ &&
+    parseInt(values.endTime.endTimeValue.selected_option.value) <
```

Comment:
I think that a nice improvement for this can be to create a util function that retrieve this value. Send as an attribute the "endTime" value. With that, will not force us to learn all this deepness and we can add some boundaries to avoid undefined values, for example